components/raftstore-v2/src/operation/ready/apply_trace.rs

```diff
@@ -627,21 +631,21 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
 
                 let flush_state = self.flush_state().clone();
                 let mut apply_trace = self.storage_mut().apply_trace_mut();
-                let mut max_flush_index = 0;
 
                 let flushed_indexes = flush_state.as_ref().flushed_index();
                 for i in 0..flushed_indexes.len() {
                     let flush_index = flushed_indexes[i].load(Ordering::SeqCst);
                     let cf = offset_to_cf(i);
                     apply_trace.on_flush(cf, flush_index);
-                    max_flush_index = u64::max(max_flush_index, flush_index);
                 }
 
-                apply_trace.maybe_advance_admin_flushed(max_flush_index);
+                apply_trace.maybe_advance_admin_flushed(applied_index);
```

Comment:
Question:  we just flushed one cf (the oldest cf), why can the flushed index be set to applied_index, as other CFs have not been flushed yet.